pkg/limits/frontend/ring.go

```diff
@@ -73,6 +74,13 @@ func newRingGatherer(
 				Help: "The total number of received streams that could not be checked.",
 			},
 		),
+		missingInstanceForPartition: promauto.With(reg).NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "loki_ingest_limits_frontend_missing_instance_for_partition_total",
```

Comment:
This is the name I had in my local branch, a bit shorter, and we can keep the naming convention for counter metrics to end with `s_total` (plural):

```suggestion
				Name: "loki_ingest_limits_frontend_missing_partitions_total",
```